bin/node/src/cli.rs

```diff
@@ -60,10 +60,6 @@ impl SubstrateCli for Cli {
         };
         Ok(Box::new(chainspec?))
     }
-
-    fn native_runtime_version(_: &Box<dyn ChainSpec>) -> &'static RuntimeVersion {
```

Comment:
What was this call used for?

Reply:
I will be honest Im not sure exactly how it was used :P

Reply:
Let's run featurnet from this branch and leave till tomorrow. If all fine then I'll resolve this conversation.